dm/syncer/data_validator.go

```diff
@@ -841,23 +851,108 @@ func (v *DataValidator) loadPersistedData(tctx *tcontext.Context) error {
 
 	v.persistHelper.setRevision(rev)
 
-	v.tableStatus, err = v.persistHelper.loadTableStatus(tctx)
+	tableStatus, err := v.persistHelper.loadTableStatus(tctx)
 	if err != nil {
 		return err
 	}
+	v.initTableStatus(tableStatus)
+	return nil
+}
+
+func (v *DataValidator) incrErrorRowCount(status pb.ValidateErrorState, cnt int) {
```

Comment:
what's the expected usage of status?

Reply:
not used now, i thought we may use this param later, so leave it. ok to remove.